README.md

```diff
@@ -22,6 +20,34 @@ You can still raise issues or pull requests if you want to. As soon as we’re a
 
 [Try out the examples!](https://alphagov.github.io/accessible-autocomplete/examples/)
 
+---
+
+## Support
+
+The GOV.UK Design System team maintains the accessible autocomplete as a standalone component. However, we’re only able to put in minimal work to support it.
+
+[Read about our plans to maintain this component.](LINK-TO-PINNED-RESEARCH-ISSUE)
+
+We provide support for:
+
+- accessibility issues that block usage
+- security issues
+- broken functionality that blocks usage for most users
+- dependency updates to keep tests working so we can make changes safely
+- documentation updates
+
+[Read more about the types of support we can provide.](LINK-TO-PINNED-SUPPORT-ISSUE)
+
+We cannot currently provide support for:
+
+- adding new features
+- smaller functionality issues that do not block usage
+- issues caused by users customising the accessible autocomplete
+- issues with browsers and assistive technologies not mentioned in the [GOV.UK Service Manual](https://www.gov.uk/service-manual/technology/designing-for-different-browsers-and-devices)
+- complex issues with React
```

Comment:
Not 100% sure about the use of 'provide support' here – possibly it implies we can only help users to fix these things, rather than fixing them ourselves? It reads slightly oddly to me – but if others are happy with it then feel free to ignore.

Reply:
@36degrees Would it clear up things up at all if we wrote "We support" and "We cannot currently support" instead?

Reply:
Just commenting to flag that this comment thread isn't outdated and we still need to decide on wording.

Reply:
Not sure it changes much TBH. I still can't articulate what's bugging me about it, so happy to be ignored on this one! 🙂

Reply:
Fair enough!